Projects/Home/Sources/Claims/ClaimDetailView.swift

```diff
@@ -79,19 +79,19 @@ public struct ClaimDetailView: View {
 
             Spacer()
         }
-        .navigationBarTitle(Text(L10n.ClaimStatus.title), displayMode: .inline)
         .onReceive(
-            store.stateSignal
-                .plain()
-                .distinct()
+            store
```

Comment:
I think this logic can be solved neater with the `PresentableStoreLens`, check here for reference: https://github.com/HedvigInsurance/ugglan/blob/666c8bc60596482e28b1db53767dcb1ef77edfeb/Projects/Contracts/Sources/View/ContractTable.swift#L35